static/app/components/charts/eventsChart.tsx

```diff
@@ -584,7 +589,7 @@ class EventsChart extends React.Component<EventsChartProps> {
 
       return (
         <TransitionChart
-          loading={loading}
+          loading={loading || loadingProcessedEventsBaseline === true}
```

Comment:
Will change it to !!loadingProcessedEventsBaseline.

Reply:
You can probably just leave it as `loadingProcessedEventsBaseline` without doing a truthy/falsy conversion, since the condition will still not pass if it is `undefined` 😄

Reply:
I believe that'll lead to a TS error like: Type 'boolean | undefined' is not assignable to type 'boolean' since that prop is optional. Put the !! just to address that. @0Calories